aspnetcore/signalr/configuration.md

```diff
@@ -278,18 +278,46 @@ Additional options for configuring timeout and keep-alive behavior are available
+| `withServerTimeout` | 30 seconds (30,000 milliseconds) | Timeout for server activity in milliseconds. If the server hasn't sent a message in this interval, the client considers the server disconnected and triggers the `Closed` event (`onclose` in JavaScript). This value must be large enough for a ping message to be sent from the server **and** received by the client within the timeout interval. The recommended value is a number at least double the server's Keep-Alive interval (`withKeepAliveInterval`) value to allow time for pings to arrive. |
 | `HandshakeTimeout` | 15 seconds | Timeout for initial server handshake. If the server doesn't send a handshake response in this interval, the client cancels the handshake and triggers the `Closed` event (`onclose` in JavaScript). This is an advanced setting that should only be modified if handshake timeout errors are occurring due to severe network latency. For more detail on the handshake process, see the [SignalR Hub Protocol Specification](https://github.com/aspnet/SignalR/blob/master/specs/HubProtocol.md). |
-| `KeepAliveInterval` | 15 seconds | Determines the interval at which the client sends ping messages. Sending any message from the client resets the timer to the start of the interval. If the client hasn't sent a message in the `ClientTimeoutInterval` set on the server, the server considers the client disconnected. |
+| `withKeepAliveInterval` | 15 seconds | Determines the interval in milliseconds at which the client sends ping messages. This setting allows the server to detect hard disconnects, such as when a client unplugs their computer from the network. Sending any message from the client resets the timer to the start of the interval. If the client hasn't sent a message in the `ClientTimeoutInterval` set on the server, the server considers the client disconnected. The ping occurs at most as often as the server pings. If the server pings every five seconds, assigning a value lower than `5000` (5 seconds) pings every five seconds. The default value is 15 seconds. The Keep-Alive interval should be less than or equal to half the value assigned to the server timeout (`withServerTimeout`). |
 
 In the .NET Client, timeout values are specified as `TimeSpan` values.
 
+In ASP.NET Core 8.0 or later <xref:Microsoft.AspNetCore.SignalR.Client.HubConnection.ServerTimeout> (default: 30 seconds) and <xref:Microsoft.AspNetCore.SignalR.Client.HubConnection.KeepAliveInterval> (default: 15 seconds) can be set directly on <xref:Microsoft.AspNetCore.SignalR.Client.HubConnectionBuilder>.
```

Comment:
Why is this needed if you are getting rid of `ServerTimeout` and `KeepAliveInterval` from the docs?

Reply:
Thanks, I can remove it.  I was a bit worried that developers already using SignalR for previous versions might be wondering what was happening there.